playback/tape_cassettes/asynchronous/async_record_only_tape_cassette.py

```diff
@@ -126,9 +126,10 @@ def _flush_recording(self):
                 _logger.exception(u"Error running recording operation - {}".format(ex))
 
 
-class AsyncRecording(Recording):
+class AsyncRecording(MemoryRecording):
```

Comment:
why will AsyncRecording inherit from MemoryRecording, maybe it is best to implement the methods in the Recording class?

Does  AsyncRecording extend MemoryRecording?

Reply:
Recording is an abstract class, the fact that the current recording implementations all inherit from MemoryRecording doesn't necessarily mean all future ones should. I am not too fond of this implementation change that I've done but I think its the simplest.
Another alternative was to remove the AsyncRecording all together and only have the save_recording asynchornous as in reality the implementations we have right now don't write anything to disk/network on the set_data and add_metadata. However a future implementor may decide to already persist stuff during the _set_data invocation and the AsyncRecording will protect against that and would make it asynchronous removing the need for him to worry about that.